src/store/trees/audio.js

```diff
@@ -43,67 +56,138 @@ export default {
     }) {
 
       commit('setAudioRecordingConfigVisibilityFor', {key, isVisible: false})
-      dispatch('showAppMessageFor', {message: 'Atempting to call...'})
-
-      return axios.post(routeFrom('trees.calltorecordStart', null, rootState.trees.ui.routes), {recorder_phonenumber, recorder_name, is_new_recorder, description})
-          .then(({data: {uuid, queue_id: queueId, status, status_description, description}}) => {
-
-            if (status_description === 'error') {
-              const message = status === 'no_credit_error' ? description : 'Error dialing number'
-              dispatch('showAppMessageFor', {message, isComplete: true})
-              return
-            }
+      // TODO: enable showAppMessageFor once available
+      // dispatch('showAppMessageFor', {message: 'Atempting to call...'})
+
+      return axios.post(routeFrom('trees.calltorecordStart', null, rootState.trees.ui.routes),
+        {recorder_phonenumber, recorder_name, is_new_recorder, description},
+        {headers: { 'Content-Type': 'multipart/form-data' }}
+          ).then(({data: {uuid, queue_id: queueId, status, status_description, description}}) => {
+
+            commit('addRecorder', {
+              id: `${recorder_name.replace(/[\W_]+/g, '')}-${recorder_phonenumber}`,
```

Comment:
@bzabos , pls confirm if you're ok with this `id` value. On voto5, it was generated from the backend I think

Reply:
When thinking, you may make a point to always generate this on the backend.

But I have some concerns:
- Currently voto5 doesn't return the recorder_id from backend https://bitbucket.org/voto/voto5/src/4111fb384534d413da604d0d903532194620f61f/app/Http/Controllers/V3CallToRecordsController.php#lines-170 . Shall we add a TODO to update this voto5 controller then? 
- I don't know the current voto5 behaviour when we try a call to record with new recorder: Does it update the recorders list in UI once the request is done, or does it requires a page refresh to see the new added recorder? (I cannot simulate this locally with the docker env. and coundl't try on DM)

Let's have a quick call on this

Reply:
Note after catchup:
- I will generate a fake `recorder_id` from devServer
- Brett will create a ticket or put a note somewhere to remind us we should update the voto5 startRecording endpoint to return the `recorder_id`